src/main/resources/application.yml

```diff
@@ -85,10 +85,6 @@ management:
     env:
       enabled: true
 
-sqs:
-  provider: aws
-  endpoint.region: eu-west-2
```

Comment:
This is defaulted as per https://github.com/ministryofjustice/hmpps-spring-boot-sqs#hmppssqsproperties-definitions